hack/build.sh

```diff
@@ -81,7 +83,7 @@ run() {
 
 go_fmt() {
   echo "🧹 ${S}Format"
-  go fmt ./cmd/... ./pkg/...
+  find $(echo $source_dirs) -name "*.go" -print0 | xargs --null gofmt -s -w
```

Comment:
```suggestion
  find $(echo $source_dirs) -name "*.go" -print0 | xargs -0 gofmt -s -w
```

`--null` is not known on macOs. `-0` should do the trick everywhere.